app/Exceptions/Handler.php

```diff
@@ -31,9 +31,17 @@ class Handler extends ExceptionHandler
      * @param  \Exception  $exception
      * @return void
      */
-    public function report(Exception $exception): void
+    public function report(Exception $exception): Response
```

Comment:
This no longer matches the return type in the comment above.  I do not believe the changes to this class are necessary.  Could you please elaborate on this change?